docs/en/sql-reference/statements/create/table.md

```diff
@@ -379,8 +379,8 @@ High compression levels are useful for asymmetric scenarios, like compress once,
 `DEFLATE_QPL` — [Deflate compression algorithm](https://github.com/intel/qpl) implemented by Intel® Query Processing Library. Some limitations apply:
 
 -   DEFLATE_QPL is experimental and can only be used after setting configuration parameter `allow_experimental_codecs=1`.
--   DEFLATE_QPL only works if ClickHouse was compiled with support for AVX2 or AVX512 instructions
--   DEFLATE_QPL works best if the system has a Intel® IAA (In-Memory Analytics Accelerator) offloading device
+-   DEFLATE_QPL only works if ClickHouse was compiled with support for AVX2 or AVX512 instructions. Refer to [Build Tips](https://github.com/ClickHouse/ClickHouse/tree/master/contrib/qpl-cmake/doc/build.md) for more details.
```

Comment:
The linked documentation will currently not be rendered with ClickHouse's nice standard documentation template (e.g. https://clickhouse.com/docs/en/sql-reference/statements/create/table).

What about including documents build.md and benchmark.md in ClickHouse's contribution guide? E.g. we could merge both files and include them on the same level as "Third-Party Libraries" or "Integrating Rust libraries" under the title "Building and Benchmarking Intel QPL".